eo-runtime/src/test/java/org/eolang/DataizedTest.java

```diff
@@ -76,4 +78,102 @@ public void close() throws SecurityException {
         );
     }
 
+    /**
+     * Try to datarize an EO app that calls a varargs func.
+     * @todo #414:30min Fix bug in datarizing a EO program calling a varargs func.
+     *  We reproduced by this test failing of datarizing with exception `You can't overwrite X`
+     *  when we try to call within a EO app a function that uses varags as parameter.
+     *  Now, we must fix it and disable the test below.
+     */
+    @Test
+    @Disabled
+    public void datarizesEOappCallingVarargsFunc() {
+        MatcherAssert.assertThat(
+            new Dataized(
+                new EOappCallingVarargsFunc(Phi.Φ)
+            ).take(Long.class),
+            Matchers.is(2L)
+        );
+    }
+
+    /**
+     * Main program sample.
+     * {@code
+     *  [] > app
+     *    (f 1 2 3).eq 2 > @
+     * }
+     * @since 0.22
+     */
+    private static class EOappCallingVarargsFunc extends PhDefault {
+        public EOappCallingVarargsFunc(Phi sigma) {
+            super(sigma);
+            this.add(
+                "φ",
+                new AtOnce(
+                    new AtComposite(
+                        this,
+                        (rho) -> {
+                            Phi fvar = new PhCopy(new DataizedTest.Fvarargs(rho));
+                            Phi var1 = new PhWith(
```

Comment:
@baudoliver7 you can do it shorter, just like this: `Phi var1 = new Data.ToPhi(1L)`

Reply:
@yegor256 Ah, I see